src/app.js

```diff
@@ -1 +1,30 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+async function getCountries() {
```

Comment:
Je doet nu niets met de functie `getCountries()`, behalve de uitkomst loggen in de terminal. Je kan er ook voor kiezen om de functie dan helemaal weg te halen.